src/Kernel/Context.class.st

```diff
@@ -2251,6 +2262,16 @@ Context >> unwindTo: aContext [
 			unwindBlock value ]].
 ]
 
+{ #category : #'debugger access' }
+Context >> variableNamesOf: anObject [
+
+	"Retrieve the names of the arguments and temporary variables, if any, that refer to the given object."
+
+	^ (1 to: self size)
+		  select: [ :i | (self at: i) == anObject ]
+		  thenCollect: [ :i | self home method tempNames at: i ]
```

Comment:
I think this might not work for temp variables that are allocated in a temp Vector (espcaping vars, when they are written to from a closure).

In smalltalk 80, there is a direct mapping temp var --> offset in the context, but with the closure implementation, they might be in an array that then is referenced from the context.